main.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import json
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+def get_team_id_season_id(url: str) -> list:
+    try:
+        response = requests.get(url)
+    except requests.exceptions.Timeout:
+        print("Request Timed Out.. Please Check Internet Connection Or Use A Valid Proxy..")
+        return ["-1", "-1"]
+    except requests.exceptions.TooManyRedirects:
+        print("Url Is Not Valid.. Please check the demo url in github and try url like that..")
+        return ["-1", "-1"]
+    except requests.exceptions.RequestException as e:
+        print("Error Occurred While Fetching Data ", e)
+        return ["-1", "-1"]
+
+    soup = BeautifulSoup(response.text, "html.parser")
+    all_data = str(soup.find("script", attrs={"id": "__NEXT_DATA__", 'type': 'application/json'}).contents[0])
+    clean_dict = json.loads(all_data)
+    extracted_team_id = clean_dict["props"]["pageProps"]["teamContext"]["data"]["teamId"]
+    extracted_season_id = clean_dict["props"]["pageProps"]["tracking"]["ssid"]
+    return [extracted_season_id, extracted_team_id]
+
+
+def get_player_stats_dict(url: str) -> dict:
+    response = requests.get(url)
+    return response.json()
+
+
+def get_selected_groups() -> list:
+    return ["Batting", "Pitching"]
+
+
+def get_header_list(each_subgroup: dict) -> list:
+    subgroup_header_list = []
+    for header_column in each_subgroup:
+        subgroup_header_list.append(header_column["header"])
+    return subgroup_header_list
+
+
+def get_column_data_of_one_row(each_row: dict) -> list:
+    column_value_list = []
+    for column in range(len(each_row)):
```

Comment:
Any time you find yourself looking up the index with range in python, there's probably a better way to do it. Check out enumerate instead.